app/sprinkles/account/src/Database/Migrations/v430/AddPasswordRestColumnUsersTable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * UserFrosting (http://www.userfrosting.com)
+ *
+ * @link      https://github.com/userfrosting/UserFrosting
+ * @copyright Copyright (c) 2019 Alexander Weissman
+ * @license   https://github.com/userfrosting/UserFrosting/blob/master/LICENSE.md (MIT License)
+ */
+
+namespace UserFrosting\Sprinkle\Account\Database\Migrations\v430;
+
+use Illuminate\Database\Schema\Blueprint;
+use UserFrosting\Sprinkle\Core\Database\Migration;
+
+/**
+ * Users table migration
+ * Adds a `flag_password_reset_required` column to the users table.
+ * Version 4.3.0.
+ *
+ * See https://laravel.com/docs/5.4/migrations#tables
+ *
+ * @author Amos Folz
+ */
+class AddPasswordRestColumnUsersTable extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public static $dependencies = [
+        '\UserFrosting\Sprinkle\Account\Database\Migrations\v400\GroupsTable',
+        '\UserFrosting\Sprinkle\Account\Database\Migrations\v400\UsersTable',
+        '\UserFrosting\Sprinkle\Account\Database\Migrations\v420\AddingForeignKeys',
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function up()
+    {
+        if ($this->schema->hasTable('users')) {
+            $this->schema->table('users', function (Blueprint $table) {
+                $table->boolean('flag_password_reset_required')->default(0)->comment('Set to 1 to force user to reset their password, 0 otherwise.')->after('flag_enabled');
```

Comment:
This formatting is easier to read. But as far as I can tell the common formatting throughout migrations is to use a single line for each column in the table. 

This style format is used in some controllers for DB queries and such though.

Reply:
Probably worth discussing, implemented in PHP CS (or whatever the thing we use is called), and documented in the style guide.

This can be addressed outside of this PR however, so I'm fine with this for now. #1020